packages/core/src/browser/config/getBrowserJestConfig.ts

```diff
@@ -0,0 +1,56 @@
+// used by cli `rw test` command
+
+const path = require('path')
+
+const { getPaths } = require('@redwoodjs/internal')
+
+const redwoodPaths = getPaths()
+
+const NODE_MODULES_PATH = path.join(redwoodPaths.base, 'node_modules')
+
+export function getBrowserJestConfig() {
+  return {
+    displayName: {
+      color: 'blueBright',
+      name: 'browser',
```

Comment:
Could this label reflect the side instead of the build target?

Reply:
I was thinking about that. We can't really know what the side is unless it's passed into the function I think.

At which point you might as well do `config.displayName.name = 'web'`

Do you have any ideas for how we can detect it?

Reply:
Hmmm... Good point, I think we can hardcode it for now, but we can definitely write a function that detects which side it's in at some point.